apps/src/maze/maze.js

```diff
@@ -228,12 +228,12 @@ function drawMap () {
   pegmanFadeoutAnimation.setAttribute('begin', 'indefinite');
   pegmanIcon.appendChild(pegmanFadeoutAnimation);
 
-  if (Maze.finish_ && skin.goal) {
+  if (Maze.finish_ && (skin.goalIdle || skin.goal)) {
```

Comment:
Let's just use one. It looks like goalIdle is the right thing to do. I looked at the other skins, and most don't actually have a goal.png. The one exception I found was scrat. Let's just have Scrat's config define goalIdle: 'goal.png'